src/main/java/ch/njol/skript/structures/StructFunction.java

```diff
@@ -32,7 +32,7 @@
 public class StructFunction extends PreloadingStructure {
 
 	static {
-		Skript.registerStructure(StructFunction.class, "function <.+>");
+		Skript.registerPreloadingStructure(StructFunction.class, 30, "function <.+>");
```

Comment:
'30' as a priority does not make sense to me, to be honest. Also, since StructOptions has a lower priority, this would mean that you still can't use options inside the function structure, right?

Reply:
No, I made lower numbers be parsed first

Reply:
That seems counter-intuitive but is ultimately just a matter of preference.

Reply:
I'd prefer number 1 to have most priority (given that all positive), so I chose this

Reply:
Oh and about 30 being a bit random, I thought it was a good idea to make all of the preloading structure priorities within Skript a multiple of 10, so if any addons wanted to be parsed right between two of Skript's elements, it'd be easy